src/components/10-atoms/text/CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 7.0.0
+
+Breaking change:
+- Remove variant `semibold`, because this option is not available in the styleguide
+- Remove variant `bold` use new boolean flag `bold`, because bold for all text sizes possible
+- Change attribute `variant` to `size`
+- When tags are placed inside slot(children). You should use the attribute `tagless` so that the wrapping paragraph is not rendered.
+- Now the text host element has margin bottom as specified in the styleguide. To edit margins, Custom CSS prop `--margin` can be used, or directly rewritten to `axa-text` margin.
```

Comment:
Difficult to understand as written. How about this:

Fix: make axa-text conformant to styleguide w.r.t. margin-bottom. To override margins, Custom CSS Property `--margin` can be used, or one can impose margins directly on `axa-text`.

Reply:
BTW, should the name --margin not be scoped somehow, e.g. by prefixing the custom element name??

Reply:
yes, `--text-margin` good catch!